swarm/storage/pyramid.go

```diff
@@ -494,8 +483,8 @@ func (self *PyramidChunker) prepareChunks(isAppend bool, chunkLevel [][]*TreeEnt
 		workers := self.getWorkerCount()
 		if int64(len(jobC)) > workers && workers < ChunkProcessors {
 			self.incrementWorkerCount()
-			storageWG.Add(1)
-			go self.processor(self.workerCount, jobC, chunkC, errC, quitC, storageWG)
+			// storageWG.Add(1)
```

Comment:
remove

Reply:
Fixed in f32d4ccca55c57de559174ba876b5bcc419a087e